decentraland/components/presentational/EthereumSignUpForm.js

```diff
@@ -44,6 +39,27 @@ export default class EthereumSignUpForm extends React.Component {
   }
 }
 
+export function TextInputField({ handleUsername, username }) {
```

Comment:
Maybe a more specific name like `UsernameTextInput` or `UsernameInputField`?

Reply:
Sure, good call

Reply:
I made this change - resolving this